src/pages/Governance/VotingWallet/index.tsx

```diff
@@ -57,6 +61,8 @@ export const VotingWalletUi: React.FC<VotingWalletUiProps> = ({
   delegateModelIsOpen,
   setDelegateModelIsOpen,
 }) => {
+  const { switchChain } = useAuth();
+  const voteProposalFeatureEnabled = useIsFeatureEnabled({ name: 'voteProposal' });
   const { t, Trans } = useTranslation();
   const styles = useStyles();
```

Comment:
Let's get rid of styled components completely here: you can replace the existing styles with Tailwind classes, delete the `styles.ts` file completely and remove the first line of this file (`/** @jsxImportSource @emotion/react */`).